src/openrct2/peep/Guest.cpp

```diff
@@ -5341,8 +5341,11 @@ void Guest::UpdateWalking()
                 for (int32_t container = HasEmptyContainerExtraFlag(); pos_extr < 32; pos_extr++)
                     if (container & (1u << pos_extr))
                         break;
-                RemoveItem(static_cast<ShopItem>(pos_extr + 32) + ShopItem::Photo2);
-                litterType = item_extra_litter[pos_extr];
+                if (pos_extr != 32)
```

Comment:
I'm not sure I understand this bit. Why was adding `ShopItem::Photo2` needed previously?

Reply:
It was a mistake photo 2 is the same as 32. So it was adding it twice

Reply:
OK, and do I understand correctly this adding of 32 is only temporary until these flags are merged in https://github.com/OpenRCT2/OpenRCT2/pull/13513 ?

Reply:
Yes